src/main/java/budzetDomowy/TransactionDAO.java

```diff
@@ -0,0 +1,97 @@
+package budzetDomowy;
+
+import java.sql.*;
+
+public class TransactionDAO implements DAO {
+    private static final String URL = "jdbc:mysql://localhost:3306/zad_domowe?characterEncoding=utf8&serverTimezone=UTC&useSSL=false";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "xxx";
+    private Connection connection;
+
+    public TransactionDAO() {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+            connection = DriverManager.getConnection(URL, USERNAME, PASSWORD);
+        } catch (ClassNotFoundException exception) {
+            System.err.println("Błąd przy wczytywaniu sterownika");
+            return;
+        } catch (
+                SQLException e) {
+            System.err.println("Błąd przy nawiązywaniu połączenia");
+            return;
+        }
+    }
+
+    @Override
+    public void select(int type) {
+        String query = "SELECT * FROM TRANSACTIONS WHERE TYPE=" + type;
+        try {
+            Statement statement = connection.createStatement();
```

Comment:
Pamiętaj że chcemy ZAWSZE używać `PreparedStatement` z racji tego że Twoje rozwiązanie z doklejaniem Stringa może kiedyś doprowadzić do SQL Injection. Ja wiem że w tym momecie sprawdzasz ten Type, ale nie wiesz jak to będize się zmieniać w przyszłości.